### ISSUE TITLE
Serial troll is making it quite difficult to track the merchant locations

### ISSUE THREAD

**Issue by Ssynchrotron** [Xeio/WanderLost#23]
It's sad that this kind of thing happens, but there is a serial troll in my region that keeps marking the wrong items and locations of merchants and that's causing players to lose precious minutes waiting for the voting system to stabilized until they can be sure they can travel to a merchant location to buy the items.

I'm not sure how difficult this would be to implement and how committed the dev of this tool is, but some kind of "shadowban" system based on the IP address of someone could help a lot at countering the low effort trolling that is going right now without resorting to account creation, since these people are unlikely to continue trolling if they have to go through the hassle of changing their IP addresses when they figure out they got caught. Basically whenever a certain IP got a low score, the shadowban system would make any suggestions made from that IP addresses to not be displayed to the rest of the IP addresses connected to the website.

**Comment by Xeio** [comment]
Yeah, it's a potential future thing. I need to do some back-end work to actually be able to store data long-term first. Right now the server forgets everything as soon as I restart it (such as to push an update) which is mostly fine because merchants only last 25 minutes anyway... but it does make anything like an account system or long-term tracking of bad/good actors... difficult.

**Comment by chance1039** [comment]
A potential solution could be to auto-hide any entries with over 10 downvotes.

**Comment by WaifuTrash** [comment]
This just happened tonight as well, some troll chucking in false Wei cards all the time. I'm all for a creative solution, maybe log IPs and if a certain IP got downvoted X amount of times then ban or shadowban their entry. Make it show up for them only, but it would obviously not count towards what other see. What do you guys think?

**Comment by WaifuTrash** [comment]
Such a shame, people are ruining a great community resource like this. I love this tool and site

**Comment by Xeio** [comment]
Added some database logging to at least capture the problem, still more work at least as far as blocking it. Which server(s) are you seeing this on most commonly at the moment so I can take a look tomorrow?

**Comment by Ssynchrotron** [comment]
In my case, it's in Kazeros (South America).

**Comment by WaifuTrash** [comment]
> Added some database logging to at least capture the problem, still more work at least as far as blocking it. Which server(s) are you seeing this on most commonly at the moment so I can take a look tomorrow?

Thirain, EUC

**Comment by Sabakabelta** [comment]
Regulus NA East seems to being targeted, not by one person but seems to be a few. Adding a troll and then downvoting the corrections as well.

**Comment by Xeio** [comment]
This is partially addressed, though there's some work to do to automate the process. 

Sadly it's not quite as many serial-trolls, at least not over a day and a half of data, but there were a few that should be blocked now.

**Comment by WaifuTrash** [comment]
Glad to hear! Keep up the great work :)

**Comment by Krueger222** [comment]
Man your tool is great, thx for the great job. With that said there is a guy ruining the experience for everybody else, can you ban an ip address?
It is impossible to use it anymore in my region and server, every time the merchants appear a guy populates the wrong info. every single time.
I am in Kazeros SA btw.

**Comment by Xeio** [comment]
Has this been any better/worse lately? Bans are having some effect, but there are a fair number of one-offs submitting bad data. I don't keep an active watch on anything but my server other than running a report occasionally.

**Comment by Sabakabelta** [comment]
> Has this been any better/worse lately? Bans are having some effect, but there are a fair number of one-offs submitting bad data. I don't keep an active watch on anything but my server other than running a report occasionally.

They do it randomly like the latest one at 4/15 - 8:30 EST on Regulus had a bunch of fake reports.

**Comment by ujellyx** [comment]
Asta-EUC just had a huge troll as well, multiple false Wei cards and legendary rapport entries... really annoying, especially when you have turned on notifications for specific things and each time the notification triggers.
My suggestion: Trigger notification only when a suggested entry has more than X upvotes.

**Comment by Xeio** [comment]
Yeah, that would probably be a good option to be configured per-user. Can't really make it global because different servers get different amounts of votes, and also if nobody gets the initial notification it probably won't get enough upvotes to trigger...

**Comment by Xeio** [comment]
Looks like someone also randomly decided to just spam Wei against all servers, there should be an auto block in place now for that particular behavior at least...

**Comment by rickadipa** [comment]
There is a troll posting every night Wei card in the Una (NA-East) as well.

**Comment by DasKashi** [comment]
another troll on Asta EUC constantly posting Legendary Cards on every respawn

**Comment by kyleshrader** [comment]
Setting an alert threshold may be useful. % upvotes AND/OR # of upvotes.

# Settings

- [ ] Upvote Percent Threshold: 0-100
- [ ] Number of Upvotes: 0-99

**Comment by Xeio** [comment]
Closing this as it seems to have (at least mostly) reduced from requiring accounts and some of the bans in place. If you see specific instances may want to let me know though at this point the few that happen seem to be one-offs which can probably just be ignored by setting a minimum upvote threshold.
